cloud/cloud.go

```diff
@@ -83,8 +85,13 @@ type (
 
 		Logger *zerolog.Logger
 		noauth bool
+
+		mu sync.Mutex
```

Comment:
```suggestion
                // mu guards the changes in Client attributes concurrently.
		mu sync.Mutex
```

Reply:
added this in my last PR (just want to avoid rebasing all of them, sorry)
thank you